app/views/hyrax/admin/appearances/_theme_form.html.erb

```diff
@@ -23,23 +23,7 @@
         </ul>
       </div>
       <div id="home-wireframe" class="col-sm-6">
-        <%= image_tag "themes/default_home/default_home", class: "img-responsive" %>
```

Comment:
🛑   This looks like we might have removed quite a bit of logic.

Reply:
Thanks for catching that